flutter/shell/platform/tizen/tizen_window_ecore_wl2.cc

```diff
@@ -469,6 +479,73 @@ void TizenWindowEcoreWl2::Show() {
   ecore_wl2_window_show(ecore_wl2_window_);
 }
 
+void TizenWindowEcoreWl2::UpdateFlutterCursor(const std::string& cursor_name) {
+#ifdef TV_PROFILE
+  int pointer_size = -1;
+  if (vconf_get_int(kSysMouseCursorPointerSizeVConfKey, &pointer_size) < 0) {
+    FT_LOG(Error) << "Failed to load cursor size.";
+  }
+
+  std::string input_cursor_name;
+  if (cursor_name == "basic") {
```

Comment:
nit: `kind` is the argument name passed by the framework so this should fairly make sense:
```suggestion
  std::string cursor_name;
  if (kind == "basic") {
```

Reply:
done